### PR TITLE
BZ:1978227 - Adding cookie=strict annotations to table for v3.11

### PR DESCRIPTION
**Pull request by kelbrown20**
For Version: 3.11 
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1978227

Preview: https://deploy-preview-40055--osdocs.netlify.app/openshift-enterprise/latest/architecture/networking/routes.html#route-specific-annotations

For Peer reviewers: Can you please add the "enterprise-3.11" label, thank you

### CODE REVIEW

File: architecture/networking/routes.adoc [hunk 1]
```diff
@@ -597,6 +597,7 @@ and "-". The default is the hashed internal key name for the route. |
 |`*router.openshift.io/haproxy.health.check.interval*`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`*haproxy.router.openshift.io/ip_whitelist*` | Sets a xref:whitelist[whitelist] for the route. |
 |`*haproxy.router.openshift.io/hsts_header*` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
+|`*router.openshift.io/cookie-same-site=Strict*` | Sets SameSite attribute to `Strict`, the cookie will not be sent along with requests initiated by third party websites. |
```
Comment (stevsmit) [inline architecture/networking/routes.adoc:600]: Based on what I read in the BZ, I would recommend the following: 

"Sets the `SameSite` attribute to `Strict`. This ensures that the cookie will not be sent along with requests initiated by third party websites." 

What you currently have reads as two independent clauses. It worked when the sentence started with 'When,' but the removal of it made it read oddly to me. I see why you want to start it with 'Sets' though.

**Comment by zhaozhanqi** [comment]
@lihongan ^^

File: architecture/networking/routes.adoc [hunk 2]
```diff
@@ -597,6 +597,7 @@ and "-". The default is the hashed internal key name for the route. |
 |`*router.openshift.io/haproxy.health.check.interval*`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
 |`*haproxy.router.openshift.io/ip_whitelist*` | Sets a xref:whitelist[whitelist] for the route. |
 |`*haproxy.router.openshift.io/hsts_header*` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
+|`*router.openshift.io/cookie-same-site=Strict*` | Sets `SameSite` attribute to `Strict`. This ensures that the cookie will not be sent along with requests initiated by third party websites. |
```
Comment (lihongan) [inline architecture/networking/routes.adoc:600]: the annotation name is `router.openshift.io/cookie-same-site` and `Strict` is just one of the accepted value, and another value is `Lax`.
so please follow https://docs.openshift.com/container-platform/4.9/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration to update this part.

**Comment by kelbrown20** [comment]
@lihongan I just made the changes to version 3.11, the same to those in the current docs. How does this look?

**Comment by lihongan** [comment]
LGTM.  Thank you @kelbrown20

**Pull request merged**